packages/core/src/components/ModalNew/footers/ModalFooterWizard/ModalFooterWizard.module.scss

```diff
@@ -0,0 +1,14 @@
+.primary {
+  order: 2;
+}
+
+.secondary {
+  order: 1;
+}
+
+.stepDots {
+  position: absolute;
+  left: 50%;
+  translate: -50%;
```

Comment:
transform: translate?

Reply:
it works basically, but you are right, it is not supported enough (104 according to caniuse)